rxkata/src/main/java/org/sergiiz/rxkata/CountriesService.java

```diff
@@ -31,4 +31,12 @@
 
     Single<Map<String, Long>> mapCountriesToNamePopulation(List<Country> countries);
 
+    Observable<Long> combineAndReturnTheSumOfPopulation(Observable<Country> countryObservable1,
+                                                        Observable<Country> countryObservable2);
+
+    Single<Boolean> checkIfBothObservableEmitsSameItems(Observable<Country> countryObservable1,
+                                                        Observable<Country> countryObservable2);
+
+    Single<Boolean> checkIfObservableEmitsGivenItem(List<Country> countries,
```

Comment:
Rename: ifObservableEmitsGivenItem.
And I'm wondering if this case is important at all, have you faced real situations where you needed such check?